toqito/state_metrics/bures_distance.py

```diff
@@ -0,0 +1,70 @@
+import numpy as np
+
+from toqito.state_metrics import fidelity
+
+def bures_distance(rho_1: np.ndarray, rho_2: np.ndarray ) -> float:
+    r"""
+    Calculate the bures distance between the two density matrices : rho_1 and rho_2, defined
+    by:
+
+     D = sqrt(2 * (1 - fidelity(rho_1, rho_2) ) )
```

Comment:
Ideally, this should be formatted using LaTeX such as:

```
.. math::
    \sqrt{2 (1 - F(\rho_1, \rho_2)}
```

where :math:`F(\cdot)` is the fidelity function and where :math:`\rho_1` and :math:`\rho_2` are density operators.